Source/Scene/Implicit3DTileContent.js

```diff
@@ -393,10 +396,52 @@ function deriveChildTile(
     contentJsons.push(combine(contentJson, implicitTileset.contentHeaders[i]));
   }
 
-  var boundingVolume = deriveBoundingVolume(
-    implicitTileset,
-    implicitCoordinates
-  );
+  var boundingVolume;
+  var childCell;
+  if (defined(implicitTileset.boundingVolume.extensions)) {
+    var boundingVolumeS2 =
+      implicitTileset.boundingVolume.extensions["3DTILES_bounding_volume_S2"];
+    if (defined(boundingVolumeS2)) {
```

Comment:
Also what do you think: do we have enough implicit bounding volume handling code that it deserves its own file? (though this would probably be best once the two branches are merged)